src/app/admin/products-table/products-table.tsx

```diff
@@ -64,12 +64,12 @@ export function ProductsTable({ products, openEdit, getProducts }: Props) {
           </Text>
         }
       >
-        <div style={{ border: '2px solid #fa5252', padding: 10, borderRadius: 5 }}>
+        <div style={{ border: '2px solid #FA5252', padding: 10, borderRadius: 5 }}>
```

Comment:
If you need to do string formatting you can do.
```ts
`2px solid ${MainColors.RedStatic}`
```

 Notice the backticks instead of quotations.